specification.md

```diff
@@ -1349,8 +1349,10 @@ For more information, see the [Workflow Error Handling](#Workflow-Error-Handling
 | --- | --- | --- | --- |
 | name | Unique retry strategy name | string | yes |
 | delay | Time delay between retry attempts (ISO 8601 duration format) | string | no |
-| multiplier | Multiplier value by which interval increases during each attempt (ISO 8601 time format). For example: "PT3S" meaning the second attempt interval is increased by 3 seconds, the third interval by 6 seconds and so on | string | no |
 | maxAttempts | Maximum number of retry attempts. Value of 0 means no retries are performed | string or number | no |
+| maxDelay | Maximum amount of delay between retry attempts (ISO 8601 duration format) | string | no |
+| increment | Static duration which will be added to the delay between successive retries (ISO 8601 duration format) | string | no |
+| multiplier | Float value by which the delay is multiplied before each attempt. For example: "1.2" meaning that each successive delay is 20% longer than the previous delay.  For example, if delay is 'PT10S', then the delay between the first and second attempts will be 10 seconds, and the delay before the third attempt will be 12 seconds. | float or string | no |
```

Comment:
Add the decimal precision info here too (1 decimal point if you define it as such with  multipleOf)